### DISCUSSION TITLE
Turbo causes page full reload on fast refresh

### DISCUSSION THREAD

**Discussion by nahasco** [vercel/next.js#67429] [Help]
### Summary

App router is incredibly slow, particulary in dev. So I decided to give turbopack a shot and it was indeed faster at a first glance. However I realised it reloads the entire page on every change instead of fast refresh. This is very annoying when I am working on a component that is reset on reload.

**Comment by jackytank** [comment]
I got the same brother, I regret migrating from my Vite React TS to NextJS just for the sake I can't connect to SFTP server in normal React so I need NextJS/Remix API routes without introduce a dedicated backend, I chose NextJS over Remix just because Ant Design have antd next-registry
